elastic/kibana
# [Security Solution] Alert flyout - update document id in analyzer preview and same ancestry

## Summary

Address: https://github.com/elastic/kibana/issues/169373

This PR updates the use of `kibana.alert.ancestor.id` to `_id` (available in flyout context as `eventId`) in analyzer preview and alerts by ancestry. This change allows upgrade from 7.x kibana to 8.10+ to utilize analyzer preview.

No UI change introduced.

**How to test**
- Analyzer preview should match that of prior to the change
- Alert by ancestry in correlations overview (right section) and correlations tab (left section -> Insights) should match that of prior to the change
- Analyzer preview should match the analyzer viewer in alerts table


### Checklist
